overlay-ot.conf

```diff
@@ -0,0 +1,49 @@
+# Hack: Disable BT on devices which support both for now
+CONFIG_BT=n
+
+CONFIG_CPLUSPLUS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+CONFIG_REBOOT=y
+
+# Disable Proxy
+CONFIG_LWM2M_FIRMWARE_UPDATE_PULL_COAP_PROXY_SUPPORT=n
+
+# Disable IPv4
+CONFIG_NET_IPV4=n
+CONFIG_NET_CONFIG_NEED_IPV4=n
+CONFIG_NET_CONFIG_MY_IPV4_ADDR=""
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR=""
+
+# OpenThread doesn't use NS/NA
+CONFIG_NET_IPV6_NBR_CACHE=n
+CONFIG_NET_IPV6_MLD=n
+
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_NET_TX_STACK_SIZE=2048
+CONFIG_NET_RX_STACK_SIZE=5120
+
+# Disable conflicting IEEE802154 L2
+CONFIG_NET_L2_IEEE802154=n
+CONFIG_NET_L2_IEEE802154_SHELL=n
+
+CONFIG_NET_L2_OPENTHREAD=y
+
+CONFIG_NET_CONFIG_MY_IPV6_ADDR=""
+CONFIG_NET_CONFIG_PEER_IPV6_ADDR="64:ff9b::0527:53ce"
+
+# Thread by default registers quite a lot addresses.
+CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=6
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=8
+
+#CONFIG_OPENTHREAD_DEBUG=y
+#CONFIG_NET_LOG=y
```

Comment:
Same here, we've already got a debug helpers section in prj.conf which has some of these (like CONFIG_NET_LOG), why duplicate this?